scripts/globals/items/death_scythe.lua

```diff
@@ -0,0 +1,37 @@
+-----------------------------------------
+-- ID: 16777
+-- Item: Death Scythe
+-- Additional Effect: Drains HP
+-----------------------------------------
+require("scripts/globals/status");
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(player,target,damage)
+    local chance = 10;
+    if (target:getMainLvl() > player:getMainLvl()) then

```

Comment:
On melee weapons with added effects:
* on status effect: fixed chance + chance to resist 
(in which case there is no message the effect just never happens)

* on element damage: fixed chance, and fixed damage that is modified ONLY by elemental resistance 
(fire dmg vs water elemental = you see the log say you hit for zero)

Drain effects work as if they were dark element dmg.